inject.js

```diff
@@ -77,34 +89,21 @@ exports.create = function (api) {
 
   function routes(sofar = []) {
     // loc = location, an object with all the info required to specify a location
-    const routes = [
+    const r = [
       [loc => loc.page === 'chess', chessIndex],
       [loc => isChessMsg(loc), chessShow],
     ];
 
     // this stacks chess routes below routes loaded by depject so far (polite)
-    return [...sofar, ...routes];
-  }
-
-
-  function ChessContainer() {
-    const root = h('div.ssb-chess-container');
-    // root.title = location.page
-    // ? '/chess'
-    // : `/chess/${getRoot(location)}`
-
-    // root.id = api.app.sync.locationId(location)
-
-    root.title = '/chess';
-    root.id = JSON.stringify({ page: 'chess' });
-
-    return root;
+    return [...sofar, ...r];
   }
 };
 
 function locationId(location) {
-  if (location.page === 'chess') return JSON.stringify({ page: 'chess' });
-  if (isChessMsg(location)) return JSON.stringify({ page: 'chess' });
+  if (location.page === 'chess' || isChessMsg(location)) {
+    return JSON.stringify({ page: 'chess' });
+  }
+  throw new Error('Unexpected location ID: ', location);
```

Comment:
I think for `depject` to work, this has to return undefined if it doesn't handle the location (to signal the next handler should be tried.)

This stops patchbay starting as the error is thrown immediately.